apps/cookbook/src/app/examples/toggle-button-example/toggle-button-example.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cookbook-toggle-button-example',
+  templateUrl: './toggle-button-example.component.html',
```

Comment:
We expose the template through a config object now - e.g. see apps\cookbook\src\app\examples\segmented-control-example\default\default.ts